mokapot/model.py

```diff
@@ -50,9 +50,10 @@ class Model:
 
     Parameters
     ----------
-    estimator : classifier object, optional
+    estimator : classifier object
         A classifier that is assumed to implement the scikit-learn
-        estimator interface.
+        estimator interface. To emulate the Percolator estimator (an
+        SVM model) use :py:class:`PercolatorModel`.
```

Comment:
suggestion: It's not clear that this should be called instead of `mokapot.Model()`

I would it change it to: 
```
To emulate Percolator (an SVM model), use :py:class:`PercolatorModel` instead.
```